Tests/Autogenerated/_BluetoothError.generated.swift

```diff
@@ -87,18 +90,22 @@ extension _BluetoothError: CustomStringConvertible {
             connection (previously establishConnection subscription was not disposed).
             """
         case let .peripheralConnectionFailed(_, err):
-            return "Connection error has occured: \(err?.localizedDescription ?? "-")"
+            return "Connection failed with error: \(err?.localizedDescription ?? "-")"
         case let .peripheralDisconnected(_, err):
-            return "Connection error has occured: \(err?.localizedDescription ?? "-")"
+            return "_Peripheral disconnected with error: \(err?.localizedDescription ?? "-")"
```

Comment:
why starting with "_"? it's this the class/struct name?

Reply:
Yes, all generated classes are prefixed with '_', they are used for testing purposes and excluded from the main target